src/Form/Control/Lookup.php

```diff
@@ -202,7 +202,7 @@ public function getData($limit = true): array
      */
     public function renderRow(\atk4\data\Model $row): array
     {
-        $renderRowFunction = is_callable($this->renderRowFunction) ? $this->renderRowFunction : [__CLASS__, 'defaultRenderRow'];
+        $renderRowFunction = $this->renderRowFunction ?? \Closure::fromCallable([static::class, 'defaultRenderRow']);
 
         return call_user_func($renderRowFunction, $this, $row);
```

Comment:
`return $renderRowFunction($this, $row);`?

Reply:
I prefer to pass args only once to be consistent